nms/app/packages/magmalte/app/components/DataGrid.js

```diff
@@ -222,57 +223,61 @@ export default function DataGrid(props: Props) {
-            zeroMinWidth
-            className={classes.dataBlock}>
-            <Grid item xs={12}>
-              {data.collapse !== undefined && data.collapse !== false ? (
-                DataCollapse(data.category, data.value, data.collapse)
-              ) : (
-                <CardHeader
-                  data-testid={data.category}
-                  className={classes.dataBox}
-                  title={data.category}
-                  titleTypographyProps={{
-                    variant: 'caption',
-                    className: classes.dataLabel,
-                    title: data.category,
-                  }}
-                  subheaderTypographyProps={{
-                    variant: 'body1',
-                    className:
-                      data.obscure === true
-                        ? classes.dataObscuredValue
-                        : classes.dataValue,
-                    title: data.value + (data.unit ?? ''),
-                  }}
-                  subheader={
-                    data.statusCircle === true
-                      ? StatusIndicator(
-                          data.statusInactive || false,
-                          data.status || false,
-                          data.value + (data.unit ?? ''),
-                        )
-                      : data.icon
-                      ? DataIcon(data.icon, data.value + (data.unit ?? ''))
-                      : data.obscure === true
-                      ? DataObscure(data.value, data.category)
-                      : data.value + (data.unit ?? '')
-                  }
-                />
-              )}
+      {row.map((data, j) => {
+        const dataEntryValue = data.value + (data.unit ?? '');
+
+        return (
+          <React.Fragment key={`data-${i}-${j}`}>
+            <Grid
+              item
+              container
+              alignItems="center"
+              xs={12}
+              md
+              key={`data-${i}-${j}`}
+              zeroMinWidth
+              className={classes.dataBlock}>
+              <Grid item xs={12}>
+                {data.collapse !== undefined && data.collapse !== false ? (
+                  DataCollapse(data.category, data.value, data.collapse)
+                ) : (
+                  <CardHeader
+                    data-testid={data.category}
+                    className={classes.dataBox}
+                    title={data.category}
+                    titleTypographyProps={{
+                      variant: 'caption',
+                      className: classes.dataLabel,
+                      title: data.category,
+                    }}
+                    subheaderTypographyProps={{
+                      variant: 'body1',
+                      className:
+                        data.obscure === true
+                          ? classes.dataObscuredValue
+                          : classes.dataValue,
+                      title: data.tooltip ?? dataEntryValue,
```

Comment:
minor nit: you can just do data.tooltip ?? data.value + (data.unit ?? '') and remove dataEntryValue. That way you can retain the shorthand notation.